content/zh/docs/tutorials/stateful-application/mysql-wordpress-persistent-volume.md

```diff
@@ -453,10 +453,10 @@ Do not leave your WordPress installation on this page. If another user finds it,
 
 {{% capture whatsnext %}}
 
-* Learn more about [Introspection and Debugging](zh/docs/tasks/debug-application-cluster/debug-application-introspection/)
-* Learn more about [Jobs](zh/docs/concepts/workloads/controllers/jobs-run-to-completion/)
-* Learn more about [Port Forwarding](zh/docs/tasks/access-application-cluster/port-forward-access-application-cluster/)
-* Learn how to [Get a Shell to a Container](zh/docs/tasks/debug-application-cluster/get-shell-running-container/)
+* Learn more about [Introspection and Debugging](/zh/docs/tasks/debug-application-cluster/debug-application-introspection/)
+* Learn more about [Jobs](/zh/docs/concepts/workloads/controllers/jobs-run-to-completion/)
```

Comment:
no. .... these should not be changed.

Reply:
why?

Reply:
they are the English original text, why change them?

Reply:
First of all, adding prefixes in the zh directory without much impact.
Secondly, it is convenient to replace.
The situation to consider for url replacement is complicated, and I have to reconsider.
I have to reconsider it. 
I remember that the Chinese link of istio.io introduced the ci mechanism.
/hold
I need to check it carefully

Reply:
The reason I am suggesting leaving the English content untouched is for future. When the upstream English content is changed again, someone will step up and do the comparison, i.e. what has changed and what hasn't. If we change the English content in the comment, comparison for difference would become a nightmare. Agreed?

Reply:
see https://github.com/k8smeetup/website-tasks/issues/3699 

Here is a direct comparison of the differences between different branches of the original English, but adding the prefix /zh directly to the English in the comment is also not a good way to handle it.
It's a bit tricky to handle.

Reply:
So ... please consider change back all the links modified in the commented out English originals. It causes more problem than provide any convenience.

Reply:
sure .